eureka-client/src/main/java/com/netflix/discovery/converters/jackson/EurekaJsonJacksonCodec.java

```diff
@@ -37,7 +39,7 @@ public EurekaJsonJacksonCodec() {
     public EurekaJsonJacksonCodec(final KeyFormatter keyFormatter, boolean compact) {
```

Comment:
I wouldn't say this is public API. All codec stuff is internal
implementation until we provide "official" codec API.

On Wed, Sep 23, 2015 at 1:40 PM, David Liu <notifications@github.com> wrote:

> In
> eureka-client/src/main/java/com/netflix/discovery/converters/jackson/EurekaJsonJacksonCodec.java
> <https://github.com/Netflix/eureka/pull/650#discussion_r40254790>:
>
> > @@ -37,7 +39,7 @@ public EurekaJsonJacksonCodec() {
> >      public EurekaJsonJacksonCodec(final KeyFormatter keyFormatter, boolean compact) {
>
> I can imaging potential future cases where we may have multiple different
> compact types. As this is exposed as public API, can we change this from a
> boolean to a more flexible type?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Netflix/eureka/pull/650/files#r40254790>.
>